### PR TITLE
ST-1458 - Hover/transition and other changes

### PR DESCRIPTION
**Pull request by sadanandl**
Hover/transition and other changes for showcase

### CODE REVIEW

**Comment by sagarb90** [comment]
Please fix the Lint issues

**Comment by sadanandl** [comment]
> Please fix the Lint issues

Fixed

**Review by sagarb90 — APPROVED** [review]

File: src/grid/style.scss [hunk 1]
```diff
@@ -223,3 +229,16 @@
...
+}
+.stc-grid-item-blur {
+    background: rgba(255, 255, 255, 0.8);
+    z-index: 2;
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+	opacity: 0;
```
Comment (amithire1201) [inline src/grid/style.scss]: indentation

File: src/grid/style.scss [hunk 2]
```diff
@@ -223,3 +229,16 @@
...
+    position: relative;
+}
+.stc-grid-item-blur {
+    background: rgba(255, 255, 255, 0.8);
+    z-index: 2;
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
```
Comment (amithire1201) [inline src/grid/style.scss]:
```suggestion
 inset: 0;
```

**Review by amithire1201 — APPROVED** [review]

**Pull request merged**